support/deploy/deploy.sh

```diff
@@ -1,10 +1,60 @@
-if [[ "${DEPLOY}" == "true" ]]; then
-  node output/deployment/deployContracts.js || exit 1
+echo "Deploying Augur to $ETHEREUM_NETWORK"
```

Comment:
Hmm, I thought the point of this PR was to get rid of these shell scripts, rather than add to them?

Reply:
We need this one entrypoint shell script to dispatch into Docker or Local deployment. Past that the goal will be to move everything into TS.

Reasoning being, if it was in TS then you'd need to have the TS built in order to run the code that would invoke the docker image -- thereby requiring a working dev env, which defeats the purpose of the Dockerizedness.